src/test/java/de/uos/informatik/ko/rcp/geneticalgorithm/GeneticAlgorithmTest.java

```diff
@@ -62,10 +62,10 @@ public static void main (String[] args){
          */
 
         //Reproduktion testen
-        reproduziert = GeneticAlgorithm.reproduktion(pop, instance,random,0.5);
-        for (int i = 0; i < instance.n(); i++) {
-            System.out.print(reproduziert[i] + " ");
-        }
+        //reproduziert = GeneticAlgorithm.reproduktion(pop, instance,random,0.5);
+        //for (int i = 0; i < instance.n(); i++) {
+        //    System.out.print(reproduziert[i] + " ");
+        //}
```

Comment:
Is this supposed to be commented out?

Reply:
I had to do this. 
I think the version, in the master branch is not correct.
The method reproduktion somehow changed.
Since i wanted to use mvn package i had to do this.

Reply:
Ah, so this part is the reason why it fails to build :+1:
Maybe we should just delete this file altogether?

Reply:
Yes, i think we could delete it.

Reply:
Want me to do it? I think it's fine to merge after :+1: